api/src/main/java/org/ehcache/spi/persistence/PersistableResourceService.java

```diff
@@ -65,6 +64,22 @@
   PersistenceSpaceIdentifier create(String name, CacheConfiguration<?, ?> config) throws CachePersistenceException;
 
   /**
+   * Creates a named {@link StateRepository state repository} in the context of the given
```

Comment:
Create seems like the wrong verb here... the semantics seem to be "Get or create"?